spec/models/solidus_subscriptions/subscription_spec.rb

```diff
@@ -155,6 +155,865 @@
     end
   end
 
+  describe '#pause' do
+    context 'when an active subscription is paused' do
+      it 'sets the paused column to true' do
+        subscription = create(
+          :subscription,
+          :actionable,
+          :with_shipping_address,
+          state: 'active',
+          paused: false
+        )
```

Comment:
We could knock off a significant amount of lines if we convert creating a subscription to a single line method call and pass the `state` and `paused` as keyword arguments. This will improve readability too :)

```ruby
      subscription = create_subscription(state: 'active', paused: false)

      def create_subscription(state:, paused:)
        create(
          :subscription,
          :actionable,
          :with_shipping_address,
          state: state,
          paused: paused
        )
      end
```

Reply:
I agree that there's a lot of duplication/lines from creating the subscription, but I think keeping the verbosity will help make the test more clear. I prefer duplicating the instantiation like that to more easily see the cause and effect of the fixture and expectations 😃

Reply:
Makes sense! It boils down to preferences in the end 😃